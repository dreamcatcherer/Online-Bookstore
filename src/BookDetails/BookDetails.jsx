import React from "react";
import styles from "./BookDetails.module.css";

const BookDetails = (props)=>{
	return (
	<div className = {styles.details}>
		  <h2>Book Details</h2>
		
		<div className={styles.detailsContainer}>
			<p>Author: {props.author}</p>
        	<p>Pages: {props.pages}</p>
        	<p>Language: {props.language}</p>
        	<p>ISBN: {props.ISBN}</p>
        	<p>Publisher: {props.publisher}</p>
        	

        </div>
	</div>
	);
};

export default BookDetails;