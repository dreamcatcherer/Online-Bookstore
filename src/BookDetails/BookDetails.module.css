
.details {
  width: 580px;
  height: 280px;
  margin: 10px;
  border-radius: 15px;
  background-color: #ffb6b3;
  text-align: center;
  padding: 20px;
  font-size: 15px;
  color: #4d1f00;
  font-family: 'Balsamiq Sans', cursive;
  box-shadow: 5px 5px 10px #ffcccc;
  border: 5px solid #ff9999;
}

.link {
  font-size: 16px;
}

.link:hover {
  font-size: 20px;
  color: #990000;
}