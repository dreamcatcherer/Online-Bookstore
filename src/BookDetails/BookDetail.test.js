import React from "react";
import { render } from "@testing-library/react";
import BookDetails from "./BookDetails.jsx";

it("renders the BookCoverImage component correctly", () => {
  const { container } = render(
    <BookDetails
      pages="28"
      weight="10.9 ounces"
      is10num="0763644765"
      is13num="978-0763644765"
      dimensions="7.06 x 0.58 x 7.06 inches"
      publisher="Candlewick; Illustrated edition (September 14, 2010)"
      readingLevel="Baby - 2 years"
      language="English"
      amazon="https://www.amazon.com/gp/product/B007HOOAH6/"
    />
  );
  expect(container).toMatchSnapshot();
});
