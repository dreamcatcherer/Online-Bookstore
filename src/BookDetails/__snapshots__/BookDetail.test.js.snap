// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the BookCoverImage component correctly 1`] = `
<div>
  <div
    class="details"
  >
    <h2>
      Book Details
    </h2>
    <div
      class="detailsContainer"
    >
      <p>
        Author: 
      </p>
      <p>
        Pages: 
        28
      </p>
      <p>
        Language: 
        English
      </p>
      <p>
        ISBN: 
      </p>
      <p>
        Publisher: 
        Candlewick; Illustrated edition (September 14, 2010)
      </p>
    </div>
  </div>
</div>
`;
